cargo/src/bin/cargo_vendor.rs

```diff
@@ -7,24 +7,6 @@
 // file, You can obtain one at https://mozilla.org/MPL/2.0/.
 // SPDX-License-Identifier: MPL-2.0
 
-#![deny(warnings)]
-#![warn(unused_extern_crates)]
-// Enable some groups of clippy lints.
```

Comment:
You don't have to remove these, just comment out "deny warnings" and it unblocks you.

Reply:
ooooh :0